src/builder/command.rs

```diff
@@ -1744,7 +1744,6 @@ impl Command {
     /// Valid tags are:
     ///
     ///   * `{name}`                - Display name for the (sub-)command.
-    ///   * `{bin}`                 - Binary name.
```

Comment:
The original issue is tagged as a breaking change [which is reserved for the next major release](https://github.com/clap-rs/clap/blob/master/CONTRIBUTING.md#compatibility-expectations).

What you can do
- Add a `(deprecated)` suffix in this help string (this must remain documented until removal so people understand what changes are made to their help)
- Conditionally remove support from it via `#[cfg(not(feature = unstable-v5))]` from `help_template` (which is missing in this PR)
- Conditionally (`not(feature = unstable-v5)` again) assert that the help template doesn't reference `{bin}` in `debug_asserts` (which is missing in this PR

Reply:
Thank you for your time and comments!

I have addressed your comments at https://github.com/clap-rs/clap/pull/4566/commits/e294f7804844d80253e6a1393fb6461e09d700a7 and https://github.com/clap-rs/clap/pull/4566/commits/c16153a3135f614026b8f487c99a89b1a236b368 .

Is my commit able to address what you have pointed out?

I am not very good at English. Sorry if I am wrong.

Reply:
Thanks!

The only thing missing is in `debug_asserts.rs`, if people have enabled `unstable-v5`, we should assert if they use `{bin}`

Reply:
I appreciate your comment.

I added assertion to `debug_asserts.rs`  in https://github.com/clap-rs/clap/pull/4566/commits/b72ad5892c30f10f5bf843a8a46ae8d99d5c8a0c

Reply:
Thanks!